docs/modules/record-encryption/hashicorp-vault/con-vault-plugin-configuration.adoc

```diff
@@ -23,5 +24,7 @@ kmsConfig:
 <3> (Optional) TLS trust configuration.
 <4> File containing the Vault Token.
 
-For TLS trust and TLS client authentication configuration, the filter accepts the same TLS parameters as xref:con-deploying-upstream-tls-{context}[Upstream TLS]
-except the `PEM` store type is currently https://github.com/kroxylicious/kroxylicious/issues/933[not supported].
```

Comment:
While it fixes the problem of the dangling ref, it does mean we lose the call to action for reporting interest in PEM...